client/src/controllers/w_keyboard_controller.ts

```diff
@@ -0,0 +1,23 @@
+import { Controller } from '@hotwired/stimulus';
```

Comment:
Best to follow the convention of all other files in the folder. 

This file should be named `KeyboardController.ts` using UpperCamelCase.ts without the `w` at the start. The `w-` is just for the identifier to help differentiate Wagtail (w) from any other custom controllers used by customisations.